client/src/js/updates/components/__tests__/Release.test.js

```diff
@@ -1,28 +1,18 @@
-import { Col } from "react-bootstrap";
-import Release, { ReleaseMarkdown } from "../Release";
+import Release from "../Release";
 
 describe("<Release />", () => {
-    let props;
-    let wrapper;
+    const props = { name: "foo", body: "bar", html_url: "baz" };
+    const state = { in: false };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5026542)